.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [published]
+jobs:
+  build-and-publish-to-pypi:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install pypa/build
+        run: >-
+          python -m
+          pip install
+          build
+          --user
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          build
+          --sdist
+          --wheel
+          --outdir dist/
+          .
+      - name: Publish distribution 📦 to PyPI
+        if: startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
```

Comment:
Do we want to follow the example 

```
if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```
Or this is intentional?

Also, any reason to use `pypa/gh-action-pypi-publish@master` instead of a particular release tag?

Reply:
Short answer: I have no idea what I am doing and am copying https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/.

I don't want anything to be triggered on a push. I only want to publish to PyPI after I finish manually creating a release.

Reply:
ah yeah, I see. That's good then. Haven't seen use of other events beside `pull_request` and `push`, but it looks right.